src/content/enterprise/administration/configuration.mdx

```diff
@@ -530,6 +531,16 @@ devStorageClass:
 
 There is only one exception where this storage class is overwritten. In case of having [volume snapshots feature](/docs/enterprise/administration/volume-snapshots/) configured, if a storage class is required for the snapshots that storage class will have preference.
 
+### globalClusterRole
+
+Okteto assings this cluster role to every user via a cluster role binding (default: ``).
```

Comment:
I think the "default: ``"  empty is confusing, just remove it since the note is very clear.